pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: lint_quido
-version: 1.0.0
+version: 1.0.1
```

Comment:
I would give that version 1.10.0 to somewhat indicate also connection to Flutter 3.10 ver.